### PR TITLE
Sushila

### PR DESCRIPTION
**Pull request by uaryal02**

### CODE REVIEW

File: AdhocTicket.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
+package bcccp.tickets.adhoc;
+
+import java.util.Date;
+
+public class AdhocTicket implements IAdhocTicket {
+	
+	public String carparkId;
```
Comment (uaryal02) [inline AdhocTicket.java]: These variables need to remain private but variable name need to be changed

File: AdhocTicketDAO.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+package bcccp.tickets.adhoc;
+
+import java.util.List;
+
+public class AdhocTicketDAO  implements IAdhocTicketDAO  {
+        
+        public String carkparkId;
```
Comment (uaryal02) [inline AdhocTicketDAO.java]: same as above

File: IAdhocTicket1.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+public interface IAdhocTicket {
+
+	public int getTicketNo();
+	public String getBarcode();
+	public String getCarparkId();
+
+	public void enter(long dateTime);
+	public long getEntryDateTime();
+	public boolean isCurrent();
```
Comment (uaryal02) [inline IAdhocTicket1.java:11]: correct boolean variable names. Well done.

**Review by uaryal02 — COMMENTED** [review]
Please make the required changes by reading the comments.

File: AdhocTicket.java (hunk 1 above)
Comment (skhati01) [reply AdhocTicket.java]: Okay

**Comment by Sarthakjo** [comment]
All the issues have been resolved and merged with master

**Review by Sarthakjo — COMMENTED** [review]
well done! all necessary changes have been made

**Pull request merged**